treehouses/remote
# system: network topology (fixes #1072)

Added a new fragment under `System` to show network topology 

How to test it:

1. Run `treehouses upgrade cli`
2. Run the remote app under `network-topology` branch
3. Connect to your Pi
4. Go to `System` fragment
5. Tap on Discover and it will show text under it
6. Tap on the text under Discover
7. Wait until the icons show up
8. Test and see if it shows all the devices in the network and that the gateway information is accurate
